### ISSUE TITLE
Representation error with starred variants of cvSidebar*/cvMainContent*

### ISSUE THREAD

**Issue by JanWaldhorn** [opieters/limecv#28]
I find the mirrored layout option very interesting, especially with regard to a two-page resume like I have.

Unfortunately, not everything is mirrored. For example, the green bar appears on the right, but the text and images remain on the left. Also, the title line would have to end at the green area (and possibly go all the way to the left).

Attached is a MWE in the hope that it helps. In my eyes a "Nice to Have" and there are probably more "urgent" issues, but I wanted to mention it.

```
% !TEX TS-program = xelatex
% !TEX encoding = UTF-8 Unicode
\documentclass[a4paper]{limecv}
\usepackage[margin=\cvMargin,noheadfoot]{geometry}
\begin{document}
\cvSetLanguage{english}
\begin{cvSidear*}
  \cvID{John}{Doe}{picture}{position}
	\begin{cvProfile}
		Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ullamcorper euismod lorem nec eleifend. Suspendisse ac varius quam. Etiam laoreet nunc orci, vestibulum imperdiet enim elementum at. Duis dictum metus sapien, eu blandit quam malesuada et. Ut viverra maximus eros.
	\end{cvProfile}
\end{cvSidear*}
\begin{cvMainContent*}
	\begin{cvEducation}
		\cvItem{Evening class: Chinese\\
			Some School, City. September 2015 -- June 2016\\
			Achieved A2 language skill in Chinese (Mandarin).}
	\end{cvEducation}
\end{cvMainContent*}
\end{document}
```

![mwe-xelatex_pub2](https://user-images.githubusercontent.com/57268762/124006012-08e11a00-d9da-11eb-956a-bc9d7a897d8e.jpg)

**Comment by JanWaldhorn** [comment]
I forgot to point out a typo. In the .cls it is called `cvSidebar` but the starred variant is missing a  **b** `cvSidear*`.

**Comment by opieters** [comment]
Thank you for pointing this out. I'll have a look at the cause.

**Comment by opieters** [comment]
This is fixed in version 0.1.12.

**Comment by JanWaldhorn** [comment]
Thank you very much for the fix. When implementing this in my document, I noticed that the fix only works if cvMainContent* comes first and then cvSidebar*. If the order is as above in the MWE, the green bar still extends into the sidebar.
However, in the correct order it works as desired.
